python/paddle/fluid/incubate/fleet/parameter_server/ir/public.py

```diff
@@ -1118,6 +1151,89 @@ def _get_optimize_ops(_program):
     return opt_ops
 
 
+def _add_lr_decay_table_pass(main_program, compiled_config, lr_decay_steps):
```

Comment:
If other decay is needed? 

how about Use a map to convert `LRScheduler` to `Variable`   to reduce code

Reply:
After simple verification, these four methods can be directly supported at present, and new methods will be added according to user requirements in the future.

This code will be continuously optimized in the future, thx~

Reply:
Ok , I think it can reuse code by dict,  too much if-else is not necessary.